pie.py

```diff
@@ -287,7 +287,10 @@ def create(self,extraArguments='',pythonCmd='python',py3=PY3):
     def cmd(self,c):
         """Runs the command `c` in this virtualenv."""
         if WINDOWS:
-            c=r'cmd /c "{}\Scripts\activate.bat && {}"'.format(self.path,c)
+            # cmd.exe /C has real specific behaviour around quotes.

```

Comment:
Interesting, I always struggled to quote complicated stuff with `cmd` and it was that simple? Nice!